src/components/AddGame/AddGame.jsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react";
+import "./AddGame.css"
+
+export default function AddGame() { //pass setter function in for allGames as prop
+  const [newGame, setNewGame] = useState({
+    title: "",
+    imagesrc: "",
+    year: "",
+    genre: [],
+    themes: [],
+    console: [],
+    developer: [],
+    publisher: [],
+  });
+
+  const handleChange = (e) => {
+    const newValue =
+      e.target.name === "year" ? parseInt(e.target.value) : e.target.value;
+    setNewGame((prevFormData) => {
+      return {
+        ...prevFormData,
+        [e.target.name]: newValue,
+      };
+    });
+  };
+
+  const handleArrayChange = (e, name) => {
+    const { value } = e.target;
+    if (value) {
+      setNewGame((prevFormData) => ({
+        ...prevFormData,
+        [name]: [...prevFormData[name], value],
+      }));
+    }
+  };
+
+//   const submitGame = (e) => {
```

Comment:
You might be able to just have this post request live in the api file where it may accept the event as a param, and just import it into your file here, it might be more findable for whoever goes looking for it in the future.

Reply:
Will do!